### PR TITLE
Spring20 repaired week03

### PR DESCRIPTION
**Pull request by neer201**
- applied [this](https://github.com/yandexdataschool/Practical_RL/issues/294)
- changed Taxi-v2 -> Taxi-v3
- moved pacman seminar and tested py3 version on MacOS

### CODE REVIEW

**Comment by dniku** [comment]
Please don't merge until https://github.com/yandexdataschool/Practical_RL/pull/322 is properly merged.

File: week03_model_free/README.md [hunk 1]
```diff
@@ -18,37 +19,8 @@
 ### Assignments
 
 Just as usual, start with 
-- `seminar_qlearning.ipynb` 
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/yandexdataschool/Practical_RL/blob/master/week03_model_free/seminar_qlearning.ipynb)
+- [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/yandexdataschool/Practical_RL/blob/master/week03_model_free/seminar_qlearning.ipynb)
```
Comment (dniku) [inline week03_model_free/README.md]: I guess that the link should be to a specific commit hash or the `spring20` branch, not `master`.

File: week03_model_free/README.md [hunk 2]
```diff
@@ -18,37 +19,8 @@
...
-python pacman.py -p <your agent> -x <number of train samples> -n <total number of samples> -l <grid env>
-python pacman.py -p PacmanQAgent -x 5000 -n 5010 -l smallGrid # example
-```
-* Make sure you can tune agent to beat ./run_crawler.sh
- * on windows, just run `python crawler.py` from cmd in the project directory
-* other ./run* files are mostly for your amusement. 
-  * ./run_pacman.sh will need more epochs to converge, see [comments](https://github.com/yandexdataschool/Practical_RL/blob/master/week03_model_free/seminar_py2/run_pacman.sh)
-  * on windows, just copy the type `python pacman.py -p PacmanQAgent -x 2000 -n 2010 -l smallGrid` in cmd from assignemnt dir
-
+- [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/yandexdataschool/Practical_RL/blob/master/week03_model_free/homework.ipynb) `homework.ipynb` _Implement EV-SARSA agent, experience replay + bonus tasks_
```
Comment (dniku) [inline week03_model_free/README.md]: Same here.

File: week03_model_free/crawler_and_pacman/seminar_py2/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Once you're done, run use those commands:
+```
+python crawler.py # Crawler with qlearning
+python pacman.py -p <your agent> -x <number of train samples> -n <total number of samples> -l <grid env>
+python pacman.py -p PacmanQAgent -x 5000 -n 5010 -l smallGrid # example
+```
+* Make sure you can tune agent to beat ./run_crawler.sh
+ * on windows, just run `python crawler.py` from cmd in the project directory
+* other ./run* files are mostly for your amusement. 
+  * ./run_pacman.sh will need more epochs to converge, see [comments](https://github.com/yandexdataschool/Practical_RL/blob/master/week03_model_free/seminar_py2/run_pacman.sh)
```
Comment (dniku) [inline week03_model_free/crawler_and_pacman/seminar_py2/README.md]: Same here.

File: week03_model_free/crawler_and_pacman/seminar_py3/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+Once you're done, run use those commands:
+```
+python crawler.py # Crawler with qlearning
+python pacman.py -p <your agent> -x <number of train samples> -n <total number of samples> -l <grid env>
+python pacman.py -p PacmanQAgent -x 5000 -n 5010 -l smallGrid # example
+```
+* Make sure you can tune agent to beat ./run_crawler.sh
+ * on windows, just run `python crawler.py` from cmd in the project directory
+* other ./run* files are mostly for your amusement. 
+  * ./run_pacman.sh will need more epochs to converge, see [comments](https://github.com/yandexdataschool/Practical_RL/blob/master/week03_model_free/seminar_py2/run_pacman.sh)
```
Comment (dniku) [inline week03_model_free/crawler_and_pacman/seminar_py3/README.md]: Same here.

File: week03_model_free/README.md [hunk 5]
```diff
@@ -2,8 +2,9 @@
 ### Materials
 * Russian materials:
    - Lecture - [video](https://yadi.sk/i/jcQ1Bg8n3SrhuQ)
-   - Q-learning seminar - [video](https://yadi.sk/i/dQmolwOy3EtGNK)
-   - Sarsa & stuff - [seminar2](https://yadi.sk/i/XbqNQmjm3ExNsq)
+   - Seminars
+      - Q-learning seminar - [video](https://yadi.sk/i/dQmolwOy3EtGNK) (older track - [assignment](https://github.com/neer201/Practical_RL/tree/spring20_repair_week03/week03_model_free/crawler_and_pacman/seminar_py2))
```
Comment (dniku) [inline week03_model_free/README.md]: The `spring20_repair_week03` will likely be deleted after this PR is merged. Please use `spring20`, or, better yet, a specific commit hash.

**Comment by neer201** [comment]
@dniku Thank you for the review!

File: week03_model_free/README.md [hunk 6]
```diff
@@ -2,8 +2,9 @@
 ### Materials
 * Russian materials:
    - Lecture - [video](https://yadi.sk/i/jcQ1Bg8n3SrhuQ)
-   - Q-learning seminar - [video](https://yadi.sk/i/dQmolwOy3EtGNK)
-   - Sarsa & stuff - [seminar2](https://yadi.sk/i/XbqNQmjm3ExNsq)
+   - Seminars
+      - Q-learning seminar - [video](https://yadi.sk/i/dQmolwOy3EtGNK) (older track - [assignment](https://github.com/neer201/Practical_RL/tree/spring20/week03_model_free/crawler_and_pacman/seminar_py2))
```
Comment (dniku) [inline week03_model_free/README.md]: Is it intended that you link to your own fork of the repo here?

**Pull request merged**